scripts/generate_dump

```diff
@@ -1005,6 +1005,31 @@ save_counter_snapshot() {
     save_cmd_all_ns "ifconfig -a" "ifconfig.counters_$idx"
 }
 
+###############################################################################
+# save the debug dump output
+###############################################################################
+save_dump_state_all_ns() {
+    IFS=';'
+    read -ra MODULES <<< "$(dump state -a)"
```

Comment:
Please use the dump state show option and use regex to parse the modules instead of introducing an option 'a' just for the purpose of tech support.